cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControlTest.java

```diff
@@ -49,19 +54,21 @@ public void testSanityCheckDryRun() throws InterruptedException, ExecutionExcept
     // For sanityCheckDryRun(false, true) and sanityCheckDryRun(false, false) (see #1 and #2 below).
     EasyMock.expect(executor.hasOngoingExecution()).andReturn(true).times(2);
     // For sanityCheckDryRun(false, XXX) (see #3 below)
+    Set<TopicPartition> dummyOngoingPartitionReassignments = new HashSet<>();
+    dummyOngoingPartitionReassignments.add(DUMMY_TOPIC_PARTITION);
     EasyMock.expect(executor.hasOngoingExecution()).andReturn(false).once();
-    EasyMock.expect(executor.hasOngoingPartitionReassignments()).andReturn(true);
+    EasyMock.expect(executor.listPartitionsBeingReassigned()).andReturn(dummyOngoingPartitionReassignments);
     // For sanityCheckDryRun(false, XXX) (see #4 below)
     EasyMock.expect(executor.hasOngoingExecution()).andReturn(false).once();
-    EasyMock.expect(executor.hasOngoingPartitionReassignments()).andReturn(false);
+    EasyMock.expect(executor.listPartitionsBeingReassigned()).andReturn(new HashSet<>());
     EasyMock.expect(executor.hasOngoingLeaderElection()).andReturn(true);
     // For sanityCheckDryRun(false, XXX) (see #5 below)
     EasyMock.expect(executor.hasOngoingExecution()).andReturn(false).once();
-    EasyMock.expect(executor.hasOngoingPartitionReassignments()).andReturn(false);
+    EasyMock.expect(executor.listPartitionsBeingReassigned()).andReturn(new HashSet<>());
```

Comment:
Nit: Since this is empty set, rather than creating a new object can we use `Collections.emptySet()`?